src/plone/restapi/tests/http-examples/translations_expand_get.resp

```diff
@@ -0,0 +1,88 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "@components": {
+        "actions": {
+            "@id": "http://localhost:55001/plone/en/test-document/@actions"
+        },
+        "aliases": {
+            "@id": "http://localhost:55001/plone/en/test-document/@aliases"
+        },
+        "breadcrumbs": {
+            "@id": "http://localhost:55001/plone/en/test-document/@breadcrumbs"
+        },
+        "contextnavigation": {
+            "@id": "http://localhost:55001/plone/en/test-document/@contextnavigation"
+        },
+        "navigation": {
+            "@id": "http://localhost:55001/plone/en/test-document/@navigation"
+        },
+        "navroot": {
+            "@id": "http://localhost:55001/plone/en/test-document/@navroot"
+        },
+        "translations": {
+            "@id": "http://localhost:55001/plone/en/test-document/@translations"
+        },
```

Comment:
An expansion of translations info is reqested. Therefore it would be good to have an example with translations. Therefore please create at least one translation.